Core/Steps/SubSteps/AddFixVersionsForNewReleaseBranchSubStep.cs

```diff
@@ -14,18 +15,22 @@ public interface IAddFixVersionsForNewReleaseBranchSubStep
 
 public class AddFixVersionsForNewReleaseBranchSubStep : AddMoveCreateFixVersionsSubStepBase, IAddFixVersionsForNewReleaseBranchSubStep
 {
+  private readonly Config _config;
+
   public AddFixVersionsForNewReleaseBranchSubStep (
+      Config config,
```

Comment:
this feels like it would be cleaner if we only injected only the String for the Project instead of the entire config that is uneeded. remember cleaning the using.